src/routes/Things.js

```diff
@@ -47,8 +49,54 @@ let router = express.Router();
  * }
  **/
 
-router.get('/Things', (req, res) => {
-  res.status(200).send();
+const resource = 'Things';
+
+router.get('/', (req, res) => {
+  db().then((dbInstance) => {
+    dbInstance.Things.findAll().then(things => {
```

Comment:
Ok, fair enough. We shouldn't retrieve fields from the DB that we are not going to use anyway.